app/models/private_childcare_provider.rb

```diff
@@ -5,15 +5,6 @@ class PrivateChildcareProvider < ApplicationRecord
 
   include PgSearch::Model
 
-  pg_search_scope :search_by_name,
```

Comment:
this is not used - and it's not even valid - there is no `name` column on the `private_childcare_providers` table, so this scope just errors.